src/api/app/controllers/webui/request_controller.rb

```diff
@@ -209,12 +209,7 @@ def accept_request
 
   def forward_request_to(fwd)
     tgt_prj, tgt_pkg = params[fwd].split('_#_') # split off 'forward_' and split into project and package
-    description = @req.description
-    description ||= ""
-    who = @req.creator
-    description += ' (forwarded request %d from %s)' % [params[:number], who]
 
-    req = nil
```

Comment:
This can't be removed, line 247 uses it and maybe is not assigned inside the transaction because an exception.